packages/phpstan-rules/src/Collector/ClassConst/ClassConstFetchCollector.php

```diff
@@ -38,6 +39,20 @@ public function processNode(Node $node, Scope $scope): ?array
         $className = $node->class->toString();
         $constantName = $node->name->toString();
 
+        $classReflection = $scope->getClassReflection();
+        if ($classReflection !== null) {
```

Comment:
the bug I ran into previously was, that we early returned `null`, when `$classReflection` was null